Templates/CSharp/EdgeDriverTemplate/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
Can this file be omitted? I don't think we need this to build the VSIX.

Reply:
Yep. Will be removed